shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -450,6 +451,13 @@ - (void)viewWillAppear:(BOOL)animated {
     _engineNeedsLaunch = NO;
   }
 
+  if (!_isAttchedView) {
```

Comment:
Wouldn't this be more appropriate in `loadView`?  Wouldn't that timing more match how it was previously and you wouldn't have to keep an extra bool `_isViewAttached`?

Reply:
Will `loadView` ever get called more than once in the lifecycle of a UIViewController?

Reply:
Talked offline - the answer should be no

Reply:
It should only be called once.  We should actually use `viewDidLoad` since it's a bit safer.

Reply:
So need also call `setViewController` in viewDidLoad. Is that right?